pages/product/[code].js

```diff
@@ -6,12 +6,84 @@ import { atomWithStorage, createJSONStorage } from "jotai/utils";
 import { useAtom } from "jotai";
 import allergens from "../../allergens.json";
 import additives from "../../additives.json";
+import { SVGIcon } from "@/components/SVGIcon";
 
 const StyledImage = styled(Image)`
+  margin: 2rem;
   width: 25%;
   height: 25%;
   object-fit: cover;
 `;
+
+const StyledProductName = styled.h2`
+  padding-top: 1rem;
+  font-weight: bold;
+  font-size: 1.2rem;
+  text-align: center;
+`;
+
+const StyledCheck = styled.div`
+  font-weight: bold;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  height: 100px;
+  width: 200px;
+  padding-top: 20px;
+  padding-right: 2rem;
+  margin-top: 2rem;
+`;
+const StyledProductCard = styled.section`
+  background-color: white;
+  margin-left: 2rem;
+  margin-right: 2rem;
+  border-radius: 0.7rem;
+  box-shadow: 0px 0px 10px 2px rgba(128, 128, 128, 0.25);
+  height: 50%;
+`;
+const StyledBackButton = styled.div`
```

Comment:
this should semantically be a button, not a div.

You will need to adapt your styling afterwards.